packages/organizations/origin-organization-irec-api/src/irec/irec.service.ts

```diff
@@ -420,7 +424,9 @@ export class IrecService implements IIrecService {
 
         const transferItem = new ReservationItem();
         transferItem.code = item.code;
-        transferItem.amount = item.volume;
+
+        // Optionally convert Wh to MWh
+        transferItem.amount = amount ? Number(amount) / 1_000_000 : item.volume;
```

Comment:
sometimes `amount` is in Wh and sometimes in MWh, wouldnt't that be confusing?

Reply:
It's in wh in this case, and the parameter is documented